packages/app/src/modules/yearOverYear.js

```diff
@@ -2,36 +2,37 @@ import i18n from '@dhis2/d2-i18n';
 
 // Fixed years generator
 const getFixedYears = len => {
-    let year = new Date().getFullYear();
-    return new Array(len)
-        .fill(null)
-        .map(n => ({ id: String(year), name: year-- }));
+    const currentYear = new Date().getFullYear();
+    return new Array(len).fill(null).map((n, i) => ({
+        id: String(currentYear - i),
+        name: () => currentYear - i,
+    }));
 };
 
 // Options for the year on year series dropdown
 export const seriesOptions = [
-    { id: 'THIS_YEAR', name: i18n.t('This year') },
-    { id: 'LAST_YEAR', name: i18n.t('Last year') },
-    { id: 'LAST_5_YEARS', name: i18n.t('Last 5 years') },
+    { id: 'THIS_YEAR', name: () => i18n.t('This year') },
```

Comment:
I know that this is just a quick fix for v33 and that most of this might have changed in later versions, but shouldn't all of these "refactor prop to function changes" also change the name of the prop? e.g. to **getName** for `name: () => i18n.t('This year')` instead of just **name.** To prevent these functions being disguised as regular props (you'd expect a prop called `name` to be used by just calling `x.name`, not `x.name()`. And `getName` to be called using `x.getName()` 🤷‍♂️ )